pkg/avatar/service.go

```diff
@@ -75,6 +75,10 @@ func (s *Service) GenerateInitials(publicName string, opts ...Options) ([]byte,
 	return bytes, s.initials.ContentType(), nil
 }
 
+func (s *Service) Shutdown(ctx context.Context) error {
```

Comment:
Don't we need to add a call to this function somewhere?

Reply:
Yep, in `cmd/serve.go`. I was working on a deeper refacto but I decided to keep this PR simple.

It appear this part of the diff have been erased with the other code. You are took quick for me, I didn't had the chance to review my PR myself :)